pyneuroml/analysis/NML2ChannelAnalysis.py

```diff
@@ -559,20 +561,23 @@ def make_iv_curve_fig(a,grid=True):
     fig.canvas.set_window_title(
         "Currents vs. holding potentials at erev = %s V"\
         % a.erev)
-    plt.xlabel('Membrane potential (V)')
-    plt.ylabel('Current (A)')
+    plt.xlabel('Membrane potential (mV)')
+    plt.ylabel('Current (pA)')
     plt.grid('on' if grid else 'off')
 
 
-def plot_iv_curve(a,hold_v,i,grid=True,same_fig=False,**plot_args):     
+def plot_iv_curve(a, hold_v, i, *plt_args, grid=True, same_fig=False, 
```

Comment:
This seems to be invalid?

Can you successfully run: **[./test.sh](https://github.com/NeuroML/pyNeuroML/blob/master/test.sh) -neuron** locally?